index.md

```diff
@@ -7,26 +7,38 @@ Storm Engine has been initially developed by studio Akella for the game [Sea Dog
 
 This knowledge base will eventually cover most aspects of the engine: engine, currently available tools, and modding. You will also find here some trivia and additional notes.
 
+For the time being, this knowledge base is only maintained in English language.
```

Comment:
I think we don't want to add a Russian version at all. it will require more effort to maintain the documentation